### PR TITLE
Development

### PR DESCRIPTION
**Pull request by rmauritsson**
Added the README.md to indicate our progress

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,22 +5,21 @@
...
         </div>
         <div class="search-area">
-            <form>
-                <input type="text" placeholder="Search">
-                <button type="submit">
-                    <img src="https://img.icons8.com/ios-glyphs/30/000000/search.png">
-                </button>
-            </form>
+                <input id="search-term" placeholder="Search" name="search_query" type="text"/>
+                <a id="search-btn" href="#">
```
Comment (Ispirett) [inline index.html]: this should be a button instead of a tag.

**Comment by Ispirett** [comment]
Hey folks really great Job with your very First project :100:  it looks so amazing :v:  Love the touch with Ariel's Photo :+1: 
There some changes and issue to fix though.
- [ ] The search button should be a button, not a  <a> tag
- [ ] Use this link  https://validator.w3.org/nu/#file and check your html file there are a lot of errors 

After changes please resubmit the project for review. You can use this same pull request link.

**Comment by Ispirett** [comment]
Also, do you have a coding partner ?

**Comment by rmauritsson** [comment]
Yes i do
His name is Kingsley McSimon

**Comment by Torres-ssf** [comment]
### Way to go team 👏
You folks sure did great on your first project. You can merge to master now. Don't forget to submit the form. Good luck with the next project.

**Comment by rmauritsson** [comment]
Thank You

**Pull request merged**